environment.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - sphinx
   - sphinx_rtd_theme
   - sphinxcontrib-apidoc
-  - sqlalchemy
+  - sqlalchemy==1.4.*
```

Comment:
Just to make sure: this is already pinned in the resulting package, so this is to make sure the development environment is consistent with it, right?

https://github.com/Quantco/pytsql/blob/e8bedc516a7cad9de908274687e494b00d1fb5eb/pyproject.toml#L39

Reply:
> Also, I propose to pin sqlalchemy to 1.4.* for now. As the integration tests are failing with 2.0 at the moment -

Then let's do `sqlalchemy<2.0` to be more explicit about the intent.
Indeed, `pytsql` is not using [SQLAlchemy 2.0 style](https://docs.sqlalchemy.org/en/14/changelog/migration_20.html) :/

E.g. it should be using `sqlalchemy.text` for raw SQL

https://github.com/Quantco/pytsql/blob/6fdf20492630b2860cca2301e0ce4e9b764d4e50/src/pytsql/tsql.py#L224-L228

Reply:
> should we than either (i) pin pyproject more explicitly within this PR as well, or (ii) separate these changes in a distinct PR?

Either is fine by me.